README.md

```diff
@@ -4,10 +4,11 @@ Contains the scripts to deploy the [Tinlake contracts](https://github.com/centri
 
 ## Requirements
 
-1. Tinlake is build and developed with  [dapp.tools](https://github.com/dapphub/dapptools) from DappHub. The deployments happens with bash scripts and the command-line tool `seth`.
+1. Tinlake is build and developed with  [dapp.tools](https://github.com/dapphub/dapptools) from DappHub. The deployments happens with bash scripts and the command-line tool `seth`. ! note: dapp.tools version has to be > 0.28.0.
 2. On MacOS, install coreutils `brew install coreutils` (this package cotains `realpath`, a dependency used in the scripts).
 3. On Linux, install `jq`, e. g. with `apt-get install jq`
 3. Init/update git submodules: `git submodule update`
+4. add env var with your etherscan API key `export ETHERSCAN_API_KEY=<<YOUR KEY>>`
```

Comment:
Maybe we can explain that they script automatically verifies the contracts on Etherscan.

However, the script should check if the the variable is not set it should not verify the contracts otherwise we might run into issues, if the script is used for deploying to a local node

Reply:
We should create a ticket.

Reply:
yes, currently local deployments are broken